.github/workflows/release.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         # publish to npm.
       - name: 🚀 PR / Publish
         id: changesets
-        uses: changesets/action@v1.5.3
+        uses: changesets/action@v1.4.10
```

Comment:
```suggestion
        # PLEASE KEEP THIS PINNED TO 1.4.10 to avoid a regression in 1.5.*
        # See https://github.com/changesets/action/issues/465
        uses: changesets/action@v1.4.10
```